src/ajax/script.js

```diff
@@ -43,15 +43,15 @@ jQuery.ajaxPrefilter( "script", function( s ) {
 // Bind script tag hack transport
 jQuery.ajaxTransport( "script", function( s ) {
 
-	// This transport only deals with cross domain requests
-	if ( s.crossDomain ) {
+	// This transport only deals with cross domain or forced-by-attrs requests
+	if ( s.crossDomain || s.scriptAttrs ) {
 		var script, callback;
 		return {
 			send: function( _, complete ) {
 				script = jQuery( "<script>" ).prop( {
```

Comment:
If `.prop(s.scriptAttrs)` is used like @jaubourg suggested maybe a single `.prop` call can be used, although probably depends on size.

Something such as:
```javascript
.prop( $.extend({}, s.scriptAttrs, {src: s.url, charset: s.scriptCharset} )
```